supabase/updateData.ts

```diff
@@ -124,7 +126,10 @@ export const saveData = async ({
 		const attachments = message.attachments.map((attachment) => ({
 			id: attachment.id,
 			message_id: message.id,
-			url: attachment.url,
+			url: attachment.url.substring(
+				attachment.url.indexOf('/attachments'),
+				attachment.url.indexOf('?')
+			),
```

Comment:
Perhaps it would be better to parse `attachment.url` in a `URL` and omit the origin?

Reply:
Make sense. Do we want to keep the params in the url?